app/main.py

```diff
@@ -1,6 +1,23 @@
+import requests
+import os
+
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+API_KEY = os.environ.get("API_KEY")
+params = {"aqi": "no", "q": CITY, "key": API_KEY}
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL, params=params).json()
```

Comment:
```suggestion
    response = requests.get(
            URL, 
            params={"aqi": "no", "q": CITY, "key": API_KEY}
     ).json()
```
and remove variable params